.prettierignore

```diff
@@ -5,3 +5,4 @@
 /src/supported-documents/*.json
 /test/mock-server/frontend
 /coverage
+/core-sdk
```

Comment:
Should be specific to `/core-sdk/FaceTec` in case of we need to introduce new things in `core-sdk` in the future. But quick question: why it's `core-sdk`? Should it be `auth-sdk` instead?

Reply:
I think the name `core-sdk` could generate confusion given we already use the term "core SDK" on mobile SDKs to refer to one of the SDK modules

Reply:
I can update the name to `auth-sdk` :) if you think it's worth it, I'll also move the files into a Facetec folder and update this line to `auth-sdk/FaceTec`